packages/salesforcedx-apex-debugger/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@salesforce/salesforcedx-apex-debugger",
+  "displayName": "Apex Debugger Adapter",
+  "description": "Implements VSCode debug protocol for Apex Debugger",
```

Comment:
Visitors to the VS Code Marketplace will probably care less about the protocol than about what the extension lets them do. Would something like "Provides integration with the Apex Debugger" work? 

I'd initially thought maybe something about providing realtime debugging for Apex code would be better, but we're going to need to make it clear that users need to have permission to use the Apex Debugger, so I don't want to promise too much in the description. 

The salesforcedx-vscode-core extension uses "Provides integration with the Salesforce CLI," so "Provides integration with the Apex Debugger" would be similar.